Dockerfile

```diff
@@ -322,8 +323,6 @@ FROM binaries AS buildkit-freebsd
 ENTRYPOINT ["/buildkitd"]
 
 FROM binaries AS buildkit-windows
```

Comment:
nit: do we remove line 324 (`FROM...`) too then?

Reply:
We aren't doing anything to the image for now but it is used [here](  https://github.com/moby/buildkit/pull/4662/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R396:~:text=FROM%20buildkit%2D%24TARGETOS%24%7BBUILDKIT_DEBUG%3A%2B%2Ddebug%7D%20AS%20buildkit) when TARGETOS is windows